src/libraries/Microsoft.NETCore.Platforms/tests/GenerateRuntimeGraphTests.cs

```diff
@@ -180,11 +180,9 @@ public void CanAddMajorVersionsToExistingGroups()
             var additionalRIDs = new[] { "rhel.10-x64" };
             var expectedAdditions = new[]
             {
-                // Note that rhel doesn't treat major versions as compatible, however we do since it's closest and we don't represent this policy in the RuntimeGroups explicitly.
-                // We could add a rule that wouldn't insert a new major version if we see existing groups are split by major version.
-                new RuntimeDescription("rhel.10", new[] { "rhel.9" }),
-                new RuntimeDescription("rhel.10-x64", new[] { "rhel.10", "rhel.9-x64" }),
-                new RuntimeDescription("rhel.10-arm64", new[] { "rhel.10", "rhel.9-arm64" })
+                new RuntimeDescription("rhel.10", new[] { "rhel" }),
+                new RuntimeDescription("rhel.10-x64", new[] { "rhel.10", "rhel-x64" }),
+                new RuntimeDescription("rhel.10-arm64", new[] { "rhel.10", "rhel-arm64" })
```

Comment:
The test changes are due combining the rhel 8 and 9 versions with TreatVersionsAsCompatible=false.

Alternatively, I can keep the split elements for 8 and 9 (without TreatVersionsAsCompatible), and only remove the minors.
That has no effect on the generated files, but the test can stay the same then.